Travis output for failed code quality tests not always informative

After #16023 if a PR fails the code quality test then the only feedback they get from Travis for their whole PR is the log for that job. Looking at this [example though](https://travis-ci.org/sympy/sympy/jobs/499396388) it seems that the output is not always very informative.

The log there ends with:
```
Listing 'sympy/vector/tests'...
Compiling 'sympy/vector/tests/__init__.py'...
Compiling 'sympy/vector/tests/test_coordsysrect.py'...
Compiling 'sympy/vector/tests/test_dyadic.py'...
Compiling 'sympy/vector/tests/test_field_functions.py'...
Compiling 'sympy/vector/tests/test_functions.py'...
Compiling 'sympy/vector/tests/test_operators.py'...
Compiling 'sympy/vector/tests/test_printing.py'...
Compiling 'sympy/vector/tests/test_vector.py'...
Compiling 'sympy/vector/vector.py'...
```
As far as I can tell there is no further output.

I don't understand why though since the script being run should be `bin/test quality` but I get more useful output from that:
```console
$ bin/test quality
====================================================== test process starts =======================================================
executable:         /Users/enojb/current/sympy/venv/bin/python  (3.7.1-final-0) [CPython]
architecture:       64-bit
cache:              yes
ground types:       gmpy 1.17
numpy:              1.16.1
random seed:        52478763
hash randomization: on (PYTHONHASHSEED=2259095863)

sympy/utilities/tests/test_code_quality.py[6] F.....                                                                        [FAIL]

__________________________________________________________________________________________________________________________________
_____________________________________ sympy/utilities/tests/test_code_quality.py:test_files ______________________________________
Traceback (most recent call last):
  File "/Users/enojb/current/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 242, in test_files
    check_directory_tree(SYMPY_PATH, test, exclude)
  File "/Users/enojb/current/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 109, in check_directory_tree
    check_files(glob(join(root, pattern)), file_check, exclusions)
  File "/Users/enojb/current/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 125, in check_files
    file_check(fname)
  File "/Users/enojb/current/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 146, in test
    test_this_file(fname, test_file)
  File "/Users/enojb/current/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 176, in test_this_file
    assert False, message_tabs % (fname, idx + 1)
AssertionError: File contains tabs instead of spaces: /Users/enojb/current/sympy/sympy/sympy/core/evalf.py, line 370.

====================================== tests finished: 5 passed, 1 failed, in 0.69 seconds =======================================
DO *NOT* COMMIT!
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -258,7 +258,7 @@ install:
       ~/.venv-no-setuptools/bin/pip uninstall -y setuptools;
       ~/.venv-no-setuptools/bin/python -We:invalid setup.py -q install;
     fi
-    python -We:invalid -m compileall -f sympy/;
+    python -We:invalid -m compileall -f -q sympy/;
     python -We:invalid setup.py -q install;
     pip list --format=columns;
 

